Configuration/python/configurationOptions.py

```diff
@@ -2603,6 +2651,9 @@
     'MET_2017D' : -1,
     'MET_2017E' : -1,
     'MET_2017F' : -1,
+    'MET_2018A' : -1,
```

Comment:
You've defined maxEvents for SingleMu_2018 and SingleEle_2018, but not for MET_2018 or Tau_2018. Maybe you want to do that?